woocommerce/woocommerce
# Add empty state to images block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





**Acceptance criteria**

- [ ] When there are no attributes, we show the empty state illustration in the Attributes block.
- [ ] The illustration is not shown on mobile.

Closes #39681.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a WooCommerce store with this branch and enable the product editor using **WooCommerce > Settings > Advanced > Features**
2. Go to **Products > Add New**
3. In the `General` tab, scroll down to the `Images` section.
4. The images place holder should be visible.
5. Add an image. The images place holder should disappear.
6. Verify the images place holder looks good on smaller screens and disappears on mobile.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
